### PR TITLE
Sprint 26 odim 3328

### PR DESCRIPTION
**Pull request by amar-shalgar**
Added the create volume logic to the svc-api and svc-systems 
Updated unit tests in the svc-api and svc-systems

Below is the unit test report for svc-api and svc-systems
/ODIM/svc-systems$ go test ./... -race
?       github.com/ODIM-Project/ODIM/svc-systems        [no test files]
ok      github.com/ODIM-Project/ODIM/svc-systems/chassis        1.315s
ok      github.com/ODIM-Project/ODIM/svc-systems/rpc    2.106s
ok      github.com/ODIM-Project/ODIM/svc-systems/scommon        121.685s
?       github.com/ODIM-Project/ODIM/svc-systems/sconstant      [no test files]
ok      github.com/ODIM-Project/ODIM/svc-systems/smodel 1.596s
?       github.com/ODIM-Project/ODIM/svc-systems/sresponse      [no test files]
ok      github.com/ODIM-Project/ODIM/svc-systems/systems        1.812s

ODIM/svc-systems$ go test ./... -count=1
?       github.com/ODIM-Project/ODIM/svc-systems        [no test files]
ok      github.com/ODIM-Project/ODIM/svc-systems/chassis        0.481s
ok      github.com/ODIM-Project/ODIM/svc-systems/rpc    0.857s
ok      github.com/ODIM-Project/ODIM/svc-systems/scommon        60.997s
?       github.com/ODIM-Project/ODIM/svc-systems/sconstant      [no test files]
ok      github.com/ODIM-Project/ODIM/svc-systems/smodel 0.926s
?       github.com/ODIM-Project/ODIM/svc-systems/sresponse      [no test files]
ok      github.com/ODIM-Project/ODIM/svc-systems/systems        0.473s

/ODIM/svc-api$ go test ./... -race
?       github.com/ODIM-Project/ODIM/svc-api    [no test files]
ok      github.com/ODIM-Project/ODIM/svc-api/handle     1.467s
?       github.com/ODIM-Project/ODIM/svc-api/messages   [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/middleware [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/models     [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/response   [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/router     [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/rpc        [no test files]
/ODIM/svc-api$ go test ./... -count=1
?       github.com/ODIM-Project/ODIM/svc-api    [no test files]
ok      github.com/ODIM-Project/ODIM/svc-api/handle     0.091s
?       github.com/ODIM-Project/ODIM/svc-api/messages   [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/middleware [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/models     [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/response   [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/router     [no test files]
?       github.com/ODIM-Project/ODIM/svc-api/rpc        [no test files]

### CODE REVIEW

File: svc-api/handle/systems_test.go [hunk 1]
```diff
@@ -475,27 +475,27 @@ func mockCreateVolume(req systemsproto.CreateVolumeRequest) (*systemsproto.Syste
...
-			StatusCode:    501, //change to 401 after logic implementation,
+			StatusCode:    401,
 			StatusMessage: "Unauthorized",
 			Body:          []byte(`{"Response":"Unauthorized"}`),
 		}
-	} else if req.SessionToken == "ValidToken" && req.SystemID == "" {
+	} else if req.SessionToken == "ValidToken" && req.StorageInstance == "" {
 		response = &systemsproto.SystemsResponse{
-			StatusCode:    501, //change to 400 after logic implementation,
+			StatusCode:    400,
```
Comment (jeevan-kamkar) [inline svc-api/handle/systems_test.go]: instead of using statuscode directly use http package

**Review by hemantha-beecherla — APPROVED** [review]
LGTM

**Review by a-ajith — APPROVED** [review]

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Please address the changes

File: svc-api/handle/systems_test.go (hunk 1 above)
Comment (amar-shalgar) [reply svc-api/handle/systems_test.go]: Addressed the comments

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by irappa-pattar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**